lib/robots/dor_repo/gis_assembly/generate_content_metadata.rb

```diff
@@ -30,112 +36,12 @@ def perform_work
             end
           end
 
-          doc = Cocina::Models::Mapping::ToMods::Description.transform(cocina_object.description, druid)
-          ns = {
-            'rdf' => 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
-            'mods' => 'http://www.loc.gov/mods/v3'
-          }
-          geo_data_xml = doc.dup.xpath('/mods:mods/mods:extension[@displayLabel="geo"]/rdf:RDF/rdf:Description', ns).first
-
-          xml = create_content_metadata(objects, geo_data_xml)
-          # metadata dir was not created upstream because no more writing geoMetadata.xml or descMetadata.xml
-          metadatadir = "#{rootdir}/metadata"
-          FileUtils.mkdir(metadatadir) unless File.directory?(metadatadir)
-          fn = "#{rootdir}/metadata/contentMetadata.xml"
-          File.binwrite(fn, xml)
-          raise "generate-content-metadata: #{bare_druid} cannot create contentMetadata: #{fn}" unless File.size?(fn)
+          object_client = Dor::Services::Client.object("druid:#{bare_druid}")
```

Comment:
Lybercore base robot already provides methods for object_client and cocina_model.

Reply:
(This is my main requested change.)

Reply:
ah yeah, this PR probably predated that, will fix

Reply:
done in third commit